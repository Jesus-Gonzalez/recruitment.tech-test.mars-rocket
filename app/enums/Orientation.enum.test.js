
const Orientation = require('./Orientation.enum');

describe('enums', () => {
    describe('Orientation', () => {
        it('should return the Orientation object', () => {
            expect(Orientation).toMatchSnapshot();
        });
    });
});
