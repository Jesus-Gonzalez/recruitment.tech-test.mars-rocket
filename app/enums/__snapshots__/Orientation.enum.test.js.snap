// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enums Orientation should return the Orientation object 1`] = `
Object {
  "East": "E",
  "North": "N",
  "South": "S",
  "West": "W",
}
`;
